src/SearchModal/SearchModal.js

```diff
@@ -227,19 +227,33 @@ class SearchModal extends Component {
               {...texts}
               data={searchResults}
               columns={columns}
-              pageSize={Math.min(pageSize, searchResults.length)}
+              pageSize={pageSize}
               loadingText={localizationTexts.loading}
               noDataText={loading ? '' : localizationTexts.noData}
               loading={loading}
               pages={pages}
               page={page}
               onPageChange={this.handlePageChange}
               onPageSizeChange={this.handlePageSizeChange}
+              getTrGroupProps={
+                (state, row) => {
+                  console.log({ row });
+                  const className = !row ? "hidden" : "";
+                  return {
+                    className,
+                  };
+                }
+              }
               getTrProps={
-                (state, row) => ({
-                  onClick: () => this.handleSelectRow(row),
-                  className: selectedRow && row && selectedRow.index === row.index ? "selected" : ""
-                })
+                (state, row) => {
+                  const onClick = () => this.handleSelectRow(row);
+                  const className = selectedRow && row && selectedRow.index === row.index ? "selected" : "null";
```

Comment:
not familiar with this className. So if it is "null" it does not do anything?

Reply:
This was garbage left from development. Modifying "null" to an empty string.